composer.json

```diff
@@ -9,16 +9,29 @@
 			"email": "hello@humanmade.com"
 		}
 	],
+	"repositories": {
+		"rlankhorst/wp-consent-level-api": {
+			"type": "vcs",
+			"url": "git@github.com:rlankhorst/wp-consent-level-api.git"
+		},
+		"altis/consent": {
+			"type": "vcs",
+			"url": "git@github.com:humanmade/altis-consent.git"
+		}
+	},
 	"require": {
 		"php": ">=7.1",
+		"altis/consent": "*",
```

Comment:
This was a placeholder for testing since the plugin itself needs an updated release. I will update that as soon as I tag it. The same applies to the vcs repository for `altis/consent` above. ☝️

Reply:
When the `altis-consent` plugin is good to go in https://github.com/humanmade/altis-consent/pull/18, I'll make a release and update this.